README.md

```diff
@@ -112,6 +112,20 @@ Finally run the Console and visit [localhost:9000](http://localhost:9000):
 ./examples/run-bridge.sh
 ```
 
+#### OpenShift (without OAuth)
```

Comment:
Should we reference these instructions from [tests](https://github.com/openshift/console#tests) / [integration tests](https://github.com/openshift/console#integration-tests)?  Only mentioning it because it sounds like (slack chat) the tests won't run w/o using this.

Reply:
Thats probably a good idea :+1: